crates/router/src/routes/admin.rs

```diff
@@ -1120,9 +1121,15 @@ pub async fn business_profiles_list(
         state,
         &req,
         merchant_id.clone(),
-        |state, _, merchant_id, _| list_business_profile(state, merchant_id),
+        |state, auth, merchant_id, _| {
+            list_business_profile(
+                state,
+                merchant_id,
+                auth.profile_id.map(|profile_id| vec![profile_id]),
+            )
+        },
```

Comment:
We shouldn't filter on profile_id in this route.

There should be a new route which filters on profile level.

Reply:
list business profile endpoint looks like this currently. `{base_url}/account/${merchantId}/business_profile`

Should the new endpoint be `{base_url}/account/${merchantId}/business_profile/profile`

Reply:
Please check with @jarnura.

Reply:
{base_url}/account/${merchantId}/profile